phpBB/phpbb/hook/finder.php

```diff
@@ -55,7 +55,9 @@ public function __construct($phpbb_root_path, $php_ext, \phpbb\cache\driver\driv
 	*/
 	public function find($cache = true)
 	{
-		if (!defined('DEBUG') && $cache && $this->cache)
+		global $phpbb_container;
+
+		if (!$phpbb_container->getParameter('finder.not_use_cache') && $cache && $this->cache)
```

Comment:
just remove the DEBUG usage inn this file

Reply:
Can the hooks be removed? i remember it was like events but from phpBB 3.0/3.1 or something like that